src/controllers/user.controller.js

```diff
@@ -181,7 +181,9 @@ exports.signin = async (req, res) => {
   if (user.password !== password) {
     return res.status(401).send({ error: true, message: INVALID_PASSWORD });
   } else {
-    const token = jwt.sign({ _id: user._id }, process.env.JWT_SECRET_KEY);
+    const token = jwt.sign({ _id: user._id }, process.env.JWT_SECRET_KEY, {
+      expiresIn: "1d",
```

Comment:
Added token expiration for 1 day (It is in testing) -- not sure it will automatically work or we need to manually check if token is expired